pyproject.toml

```diff
@@ -640,6 +640,8 @@ filterwarnings = [
     # Example Actual Warning:
     # sqlalchemy.exc.RemovedIn20Warning: Deprecated API features detected! These feature(s) are not compatible with SQLAlchemy 2.0. To prevent incompatible upgrades prior to updating applications, ensure requirements files are pinned to "sqlalchemy<2.0". Set environment variable SQLALCHEMY_WARN_20=1 to show all deprecation warnings.  Set environment variable SQLALCHEMY_SILENCE_UBER_WARNING=1 to silence this message. (Background on SQLAlchemy 2.0 at: https://sqlalche.me/e/b8d9)
     'ignore: Deprecated API features detected! These feature\(s\) are not compatible with SQLAlchemy 2\.0\.',
+    # snowflake
+    "module: The GenericFunction 'flatten' is already registered and is going to be overridden:sqlalchemy.exc.SAWarning"
```

Comment:
Did we get to the bottom of why this is happening?

Reply:
i didn't investigate, just added this so ci would run. i believe @Kilo59 looked into it this morning and has a pr to address.

Reply:
https://github.com/great-expectations/great_expectations/pull/10095#discussion_r1672196930